plugins/volume-reclaim.yaml

```
@@ -0,0 +1,56 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: volume-reclaim
```

```suggestion
  name: unused-volumes
```
`volume-reclaim` sounds like it actually does reclaim something. But it only lists volumes, right?